src/darwin/Framework/CHIP/MTRDevice_XPC.mm

```diff
@@ -309,4 +309,41 @@ - (void)downloadLogOfType:(MTRDiagnosticLogType)type
 // Not Supported via XPC
 // - (oneway void)downloadLogOfType:(MTRDiagnosticLogType)type nodeID:(NSNumber *)nodeID timeout:(NSTimeInterval)timeout completion:(void (^)(NSURL * _Nullable url, NSError * _Nullable error))completion;
 
+#pragma mark - Storage Overrides
+
+- (NSDictionary<MTRClusterPath *, MTRDeviceClusterData *> *)_clusterDataToPersistSnapshot
```

Comment:
If the goal is to not have persistence at all in MTRDevice_XPC, then we should just rip it out of MTRDevice, not do this, I think....

Reply:
Have at it :)